dbtwiz/model.py

```diff
@@ -0,0 +1,101 @@
+from enum import Enum
+from pathlib import Path
+from typing import Dict, List
+import yaml
+
+from dbtwiz.logging import fatal
+
+from .config import project_path
+
+
+def layer_choices() -> Dict[str, str]:
+    """Dict of dbt layers and descriptions"""
+    return {
+        "staging": "Initial building blocks mapping the source data",
+        "intermediate": "Logic to prepare data to be joined into products at later stages",
+        "marts": "Data products made available to several consumers",
+        "bespoke": "Data products tailored to one specific consumer",
+    }
+
+
+def materialization_choices() -> Dict[str, str]:
+    """Dict of dbt materializations and descriptions"""
+    return {
+        "ephemeral": "",
```

Comment:
Could add descriptions for each materialization type

Reply:
Yes, that's the plan, and also some way of showing them when selecting.